src/Semantic/Graph.hs

```diff
@@ -230,7 +231,7 @@ runScopeGraph :: Ord address
 runScopeGraph = raiseHandler (runState lowerBound)
 
 -- | Parse a list of files into a 'Package'.
-parsePackage :: (Member Distribute sig, Member (Error SomeException) sig, Member Resolution sig, Member Task sig, Member Trace sig, Carrier sig m)
+parsePackage :: (Member Distribute sig, Member (Error SomeException) sig, Member Resolution sig, Member Parse sig, Member Trace sig, Carrier sig m)
```

Comment:
This still requires a whole whack of effects, but _not_ telemetry, the `TaskSession`, etc., or any of the other bits implied by `Task` but not necessarily implied by `Parse`.